site/content/docs/developing/internal-traffic.en.md

```diff
@@ -32,10 +34,33 @@ endpoint := fmt.Sprintf("http://api.%s/some-request", os.Getenv("COPILOT_SERVICE
 
-When our front-end makes this request, the endpoint `api.test.kudos.local` resolves to a private IP address and is routed privately within your VPC. 
+When our front-end makes this request, the endpoint `api.test.kudos.local` resolves to a private IP address and is routed privately within your VPC.
 
-## Legacy Environments and Service Discovery
+### Legacy Environments and Service Discovery
 
 Prior to Copilot v1.9.0, the service discovery namespace used the format _{app name}.local_, without including the environment. This limitation made it impossible to deploy multiple environments in the same VPC. Any environments created with Copilot v1.9.0 and newer can share a VPC with any other environment.
 
-When your environments are upgraded, Copilot will honor the service discovery namespace that the environment was created with. That means that the endpoint your services are reachable at will not change. Any new environments created with Copilot v1.9.0 and above will use the _{env name}.{app name}.local_ format for service discovery, and can share VPCs with older environments. 
+When your environments are upgraded, Copilot will honor the service discovery namespace that the environment was created with. That means that the endpoint your services are reachable at will not change. Any new environments created with Copilot v1.9.0 and above will use the _{env name}.{app name}.local_ format for service discovery, and can share VPCs with older environments.
+
+## Service Connect
+Similar to Service Discovery, ECS Service Connect provides a “one-click” experience to have your client services connect to their dependencies in a load-balanced and resilient fashion, simplifying the way of exposing those dependencies to their clients. With Service Connect, each service you create is given a private (in-VPC) alias by default, which is also configurable in your service manifests.
+
+### How do I use Service Connect?
+Imagine we have the same app called `kudos` and two services: `api` and `front-end`, deployed in the same environment. In order to use Service Connect, both services' manifests need to have:
+
+```yaml
+network:
+  connect: true # Defaults to "false"
+```
+
+After deploying both services, they should be able to talk to each other using the default Service Connect endpoint, which is the same as its service name. For example, `front-end` service can simply call `http://api`.
+
+However, if you are already using Service Discovery right now and want to avoid any code changes, you can configure [`network.connect.alias`](../manifest/lb-web-service.en.md#network-connect-alias) field so that the Service Connect uses the same alias as Service Discovery. And if **both** client and caller have Service Connect enabled, they'll connect via Service Connect instead of Service Discovery. For example, in the manifest of the `api` service we have
```

Comment:
```suggestion

### Upgrading from Service Discovery
Prior to v1.24, Copilot enabled private service-to-service communication with [service discovery](#service-discovery). If you are already using Service Discover and want to avoid any code changes, you can configure [`network.connect.alias`](../manifest/lb-web-service.en.md#network-connect-alias) field so that the Service Connect uses the same alias as Service Discovery. And if **both** client and server have Service Connect enabled, they'll connect via Service Connect instead of Service Discovery. For example, in the manifest of the `api` service we have:
```